src/schemas/shape.ts

```diff
@@ -1,7 +1,9 @@
 import * as Yup from "yup";
 import { SchemaShape } from "./types";
 import type { TokenWithAmount as TWA } from "types/tx";
+import { OptionType } from "components/Selector";
```

Comment:
@misicnenad, everything looks good , except `schemas` now imports from `components`. Though I have not documentented this,  I've been striving to not to make cyclical imports between folders (which having one proved to be so catastrophic when trying to make `package/library` out of a folder in monorepo setup ) and keep it structured e.g

third-party
types
constants
schemas
helpers 
components
pages

though not yet perfect throughout the codebase atm
e.g. u can see `schemas/string` importing from `helpers` 

one solution could be to put `OptionType` into say `types/utils` as I have found that `slices` also imports from components currently 🤦‍♂️

Reply:
Good catch, I'd like to avoid importing from `components/*` folder inside `src/schemas/*`.

> one solution could be to put `OptionType` into say `types/utils`

Sounds good

Reply:
See:
https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2456/commits/1abe760540aafab25df6c680942cf3ff21ecea09
https://github.com/AngelProtocolFinance/angelprotocol-web-app/pull/2456/commits/539c39cdcf00621751e62519146c3cfeafe59463